dashboard/app/views/home/_survey.html.haml

```diff
@@ -1,104 +1,78 @@
-- if show_survey?
-  - survey_url = "https://docs.google.com/forms/d/10Qebl7Gi9DTf-RDt-0v78jVFdzJN0mWTSHgAuk6V5gU/viewform?entry.692842963=__other_option__&entry.692842963.other_option_response=#{current_user.id}&entry.1146050975&entry.796977513&entry.999252886"
-  .survey{style: "border: 4px solid #a69bc1; border-radius: 15px; padding:40px"}
-    #surveythanks{style: "display: none; font-size: 14px"}
-      Thank you for telling us about your students!
-      %br/
-      %br/
-      We're making changes to improve Code Studio. We'd love your feedback on how we can make it work best for your classroom.
-      %br/
-      %br/
-      %a{href: survey_url, target: "_blank"}
-        %button.btn-primary
-          Go to survey
+:css
 
+  .survey input[type=radio]:checked + label {
+    background-color: rgb(0, 178, 192) !important;
+    color: white !important;
+  }
 
-    #surveybody
-      %h1{style: "margin-top:0px; color: rgb(80,80,80)"} Code.org Program Improvement Survey
-      %div{style: "margin-left: 0px"}
-        One of our primary goals at Code.org is to improve overall diversity within computer science education. Please answer 2 questions about your classroom so we can estimate how we're doing. To protect their privacy, we will not ask your students for this information so that it is not linked to their specific accounts. (See our
```

Comment:
I haven't sorted out the details yet on how this is different but I believe the primary goals are 1) to get info about students < 13 and 2) attach a follow up google survey about professional learning and other things we want feedback on.